core/build.gradle.kts

```diff
@@ -33,7 +33,10 @@ repositories {
 
 dependencies {
     testImplementation(kotlin("test"))
-    testImplementation("org.testcontainers:testcontainers:1.19.0")
+    testImplementation("org.testcontainers:couchbase:1.20.3")
+    implementation("com.couchbase.client:kotlin-client:1.4.0")
+    implementation("org.slf4j:slf4j-api:2.0.9")
```

Comment:
do we need the slf4j-api?

Reply:
The couchbase SDK uses slf4j for logging. If we want to see the couchbase instance logs, we need to have it.
I would leave it in, what do you think?

Reply:
ok, then leave it